espnet2/asr/frontend/s3prl.py

```diff
@@ -36,75 +21,37 @@ def __init__(
         download_dir: str = None,
         multilayer_feature: bool = False,
```

Comment:
It seems this argument is presented but not used in this class.

Reply:
Right, will delete this!

Reply:
Oh, I thought you might leave this argument to keep the backward compatibility. So `multilayer_feature=True` will pass the output of S3PRLUpstream (the entire `all_hidden_states`) into Featurizer for a weighted-sum, and `multilayer_feature=False` will simply feed the last item of `all_hidden_states` (`all_hidden_states[-1:]`) into Featurizer. Feel free to delete this argument if you can change this API directly.